themes/custom/ts_wrin/sass/components/_region-map.scss

```diff
@@ -16,6 +16,12 @@
     /* line text up with border. */
     margin-top: -0.5rem;
 
+    &.margin-bottom-md {
+      @media (max-width: rem(767)) {
+        margin-bottom: 0;
```

Comment:
I realize I should have asked this first, but does doing `display:none` help performance, or would we need to do some kind of ajax-load on non-mobile to make this faster?

Reply:
I was all set to be completely wrong about this and then though better of it! It helps performance a _tiny_ bit by not rendering the map, but all of the SVG code still gets loaded as markup. After some research, and tinkering with the existing `wri_maps.js`, I've got it working nicely with a responsive `data-svg-url`  setup that retains the links. 

As a kind of bonus, on desktop this should also help performance, b/c it takes the SVG markup out of the initial page load, but without causing a repaint due to the loaded element container having an aspect-ratio set that matches the svg.